Intersect.Client.Core/Interface/Game/Spells/SpellItem.cs

```diff
@@ -288,9 +337,9 @@ public void Update()
 
                     if (bestIntersectIndex > -1)
                     {
-                        if (mYindex != bestIntersectIndex && !Globals.Me.IsCasting)
+                        if (SlotIndex != bestIntersectIndex && !Globals.Me.IsCasting)
                         {
-                            Globals.Me.SwapSpells(bestIntersectIndex, mYindex);
+                            Globals.Me.SwapSpells(bestIntersectIndex, SlotIndex);
```

Comment:
`Globals.Me` should have been cached the first time it was accessed in this method